docs/advanced/transaction-details.md

```diff
@@ -9,7 +9,7 @@ Decred transactions are transfers of DCR that exist within blocks. Transactions
 
 Field        | Description                                                                                    | Size
 ---          | ---                                                                                            | ---
-Version      | Transaction version. This number is used to signify how the transaction should be interpreted  | 4 bytes
+Version      | Transaction version. This number is used to signify how the transaction should be interpreted. The first two bytes specify the version number. The second two bytes specify the serialization format.   | 4 bytes
```

Comment:
I would say `The upper 16-bits specify the serialization format and the lower 16-bits specify the version number.`  

Whether or not it's the first or last 2 bytes depends on byte order while specifying them in terms of upper and lower bits does not.

Reply:
Changed to that wording. Though when I'm back I want to wrap my head around this bit shifting business so I can know my up from down :)